### ISSUE TITLE
Error: Cannot find module 'browserstack-config.json'

### ISSUE THREAD

**Issue by rushby** [DevExpress/testcafe-browser-provider-browserstack#152]
I have a .env file:

`BROWSERSTACK_CAPABILITIES_CONFIG_PATH="./browserstack-config.json"`

the browserstack-config.json is in the top level directory in the project, when I run the tests with this environment variable the following error is thrown:

`Error: Cannot find module './browserstack-config.json'`

Removing the environment variable successfully runs the tests, am I missing something?

**Comment by AlexKamaev** [comment]
TestCafe doesn't provide support for .env files out of the box, so I assume that you use some custom module like `dotenv` for parsing .env files. There should not be any error here, however, it's difficult to recommend anything without an example that demonstrates the issue.
Please share a basic project with your .env file and `dotenv` (or a similar module you use) which will allow us to reproduce the issue on our side.
Please also share the command that you use to run TestCafe in terminal.

**Comment by alexphilin** [comment]
Hi @AdrienLemaire

We will update our documentation and mention that a search for the `BROWSERSTACK_CAPABILITIES_CONFIG_PATH` file is run using the `require` function mechanism.

**Comment by nick-msm** [comment]
@alexphilin I don't believe the documentation was updated, I've just spent quite a bit of time debugging this issue as well. Can you commit to updating the docs please?

**Comment by Aleksey28** [comment]
Hi @nick-msm ,

Thank you for reminding. I put it in our TODO list.
